modules/market/src/main/java/com/opengamma/strata/market/curve/CurveCalibrationInfo.java

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (C) 2015 - present by OpenGamma Inc. and the OpenGamma group of companies
+ * 
+ * Please see distribution for license.
+ */
+package com.opengamma.strata.market.curve;
+
+/**
+ * Information about curve calibration.
+ * <p>
+ * Implementations of this marker interface provide information that was
+ * obtained during curve calibration, such as a Jacobian.
+ */
+public interface CurveCalibrationInfo {
```

Comment:
It is used in `NodalCurveDefinition` to allow the calibration data to be added after/separately from building the curve. I tried to remove it, but it would have locked in a type in a location that would be undesirable, limiting flexibility.